bundles/org.openhab.binding.tivo/src/main/java/org/openhab/binding/tivo/internal/service/TivoStatusProvider.java

```diff
@@ -44,6 +44,9 @@
 
 @NonNullByDefault
 public class TivoStatusProvider {
+    private static final Pattern TIVO_STATUS_PATTERN = Pattern.compile("^CH_STATUS (\\d{4}) (?:(\\d{4}))?");
+    private static final Integer READ_TIMEOUT = 3000;
```

Comment:
Primitive types (int) should be used where possible.

It's good practice to append the unit to the field name e.g. TIMEOUT_SEC.

Reply:
Fixed both and two other Integers